pandas/io/pytables.py

```diff
@@ -2661,6 +2661,8 @@ def read_index_node(self, node, start=None, stop=None):
 
         if 'name' in node._v_attrs:
             name = _ensure_str(node._v_attrs.name)
```

Comment:
you can just directly call this, no if is needed

Reply:
Done